client/blindbox/tests/session.py

```diff
@@ -1,14 +1,14 @@
 from blindbox.requests  import SecureSession, requests
 
-with SecureSession("http://localhost:8080", True) as secure_session:
+with SecureSession("http://20.223.63.77:80", "./cce_policy.txt",True) as secure_session:
```

Comment:
I am against leaving an IP in the repo. Please modify the code to specify an IP in an environment variable.